src/t8_geometry/t8_geometry_base.hxx

```diff
@@ -65,8 +65,7 @@ struct t8_geometry
    * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
    */
   virtual void
-  t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, double out_coords[3]) const
-    = 0;
+  t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, double out_coords[3]) const = 0;
```

Comment:
Please encapsulate it in a 
```
/* clang-format off */
...
/* clang-format on */
```
Did you update your pre-commit script yet? It should've catched that

Reply:
On second thaughts, we shouldn't encapsulate it in a clang-format off/on block. It is not the prettiest format, but it isn't hurting either in my opinion. Be consistant about this format would mean to add a lot off of clang-format off/on blocks, which isn't a good decision either in my opinion. What do you think?

Reply:
I pushed changes using clang-format version 16.0.1 and opend a PR to enforce version 16.0.1